Lib/socket.py

```diff
@@ -348,79 +349,86 @@ def makefile(self, mode="r", buffering=None, *,
         text.mode = mode
         return text
 
-    if hasattr(os, 'sendfile'):
+    def _sendfile_zerocopy(self, zerocopy_func, giveup_err, file, offset=0, count=None):
```

Comment:
Just to be sure, the implementation didn't change at all, just that it was now more generic right?

Reply:
Yes, it didn't